docs/data/tree-view/rich-tree-view/items/items.md

```diff
@@ -217,6 +217,19 @@ const childrenIds = apiRef.current.getItemOrderedChildrenIds(
 
 {{"demo": "ApiMethodGetItemOrderedChildrenIds.js", "defaultCodeOpen": false}}
 
+### Get an item's parent id
+
+Use the `getParentId` API method to get the id of the item's parent.
+
+```ts
+publicAPI.getParentId({
```

Comment:
```suggestion
publicAPI.getParentId(itemId);
```

The params are not an object
I removed the JSDoc because it feels 100% self-explanatory, but feel free to keep it :+1: